pkg/cmd/dcos_cluster_list.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"sync"
+
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/dcos/dcos-cli/pkg/config"
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+	"github.com/spf13/cobra"
+)
+
+// newCmdClusterList lists the clusters.
+func newCmdClusterList(ctx *cli.Context) *cobra.Command {
+	var onlyAttached bool
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:  "list",
+		Args: cobra.MaximumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			manager := ctx.ConfigManager()
+			confs := manager.All()
+
+			currentConf, err := manager.Current()
+			if err != nil {
+				return err
```

Comment:
Having no attached cluster shouldn't trigger an error, we would then just not display any star in the list.